webpack/assets/javascripts/hosts/tableCheckboxes.js

```diff
@@ -0,0 +1,251 @@
+import $ from 'jquery';
+import { sprintf, n__, translate as __ } from '../react_app/common/I18n';
+
+require('jquery.cookie');
+
+// Array contains list of host ids
+$.cookieName = `_ForemanSelected${window.location.pathname.replace(/\//, '')}`;
+$.foremanSelectedHosts = readFromCookie();
```

Comment:
I know, you are just moving it, not writing, but this assign seems especially fishy, maybe would be nice to rewrite it?
Do not understand the purpose, but assigning some property to the jquery doesn't sound like a good idea, I would suggest to put it at least in `window` in this PR.